src/crush/CrushTester.cc

```diff
@@ -503,12 +503,6 @@ int CrushTester::test()
   // make adjustments
   adjust_weights(weight);
```

Comment:
we might want to check `num_devices_active` before | after `adjust_weights` because `CrushTester::test` is expected to do sanity checks.

Reply:
@shinobu-x Thanks for your review, but I don't see any check before | after adjust_weights.